server/graphql/v2/query/collection/AccountsCollectionQuery.ts

```diff
@@ -75,7 +81,11 @@ const AccountsCollectionQuery = {
 
       // Bind arguments
       if (args.tag?.length) {
-        where['tags'] = { [Op.contains]: args.tag };
+        if (args.tagSearchOperator === 'OR') {
+          where['tags'] = { [Op.in]: args.tag };
```

Comment:
I'm surprised, `Op.in` is a way to check that a value is included in an array, but as far as I'm aware it's not meant to do an overlap between two arrays:

```sql
SELECT 'test' IN ('test', 'other') # true
```

```sql
SELECT ('test', 'test2') IN ('test', 'other') # ERROR
```

Unless I'm missing something with how Sequelize interprets this, I think we want to do something like:

```suggestion
          where['tags'] = { [Op.overlap]: args.tag };
```

Reply:
Oooh, good catch! I guess I only ran the other query while testing this.